src/app.js

```diff
@@ -0,0 +1,247 @@
+/*!
+GPII Electron
+Copyright 2016 Steven Githens
+Copyright 2016 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+The research leading to these results has received funding from the European Union's
+Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+/* eslint-env node */
```

Comment:
This directive can be placed in your eslintrc since all of your files are node-only